source/es/index.rst

```diff
@@ -0,0 +1,177 @@
+﻿.. archivo maestro de documentación de prueba, creado por
+   sphinx-quickstart el Vie Ene 28 23:39:07 2011.
+   Puede adaptar este archivo completamente a su gusto, pero al menos debería
+   contener la directiva raíz `toctree`.
+
+
+Guía de uso de InVEST
+=====================
+
+.. toctree::
+   :maxdepth: 3
+
+.. figure:: ../en/index/header.png
+
+.. figure:: ../en/index/main_image.png
+
+**Cita**: Natural Capital Project, 2022. InVEST |version|
+User’s Guide. Stanford University, University of Minnesota, Chinese Academy
+of Sciences, The Nature Conservancy, World Wildlife Fund, and Stockholm
+Resilience Centre.
+
+**Colaboradores/as**\*:
+
+.. hlist::
+   :columns: 3
+
+   * Angarita, Héctor
+   * Arkema, Katie
+   * Bhagabati, Nirmal
+   * Bailey, Allison
+   * Bernhardt, Joanna
+   * Bierbower, Will
+   * Brauman, Kate
+   * Bryant, Benjamin
+   * Cameron, David
+   * Chan, Kai
+   * Chaplin-Kramer, Rebecca
+   * Chaumont, Nicolas
+   * Conte, Marc
+   * Daily, Gretchen
+   * Davies, Jeremy
+   * Delevaux, Jade
+   * Denu, Douglas
+   * Douglass, James
+   * Eichelberger, Bradley
+   * Ennaanay, Driss
+   * Faries, Joe
+   * Fisher, David
+   * Glowinski, Kathryn
+   * Goldstein, Jesse
+   * Goldstein, Joshua
+   * Griffin, Robert
+   * Guannel, Gregory
+   * Guerry, Anne
+   * Guswa, Andrew
+   * Janke, Ben
+   * Johnson, Justin
+   * Hamel, Perrine
+   * Han, Baolong
+   * Kareiva, Peter
+   * Kennedy, Christina
+   * Kim, Choong-Ki
+   * Lacayo-Emery, Martin
+   * Leon, Jorge
+   * Lin, Chia-Yi (Joanna)
+   * Liu, Hongxiao
+   * Lonsdorf, Eric
+   * Lozano, Juan Sebastian
+   * Mandle, Lisa
+   * Mendoza, Guillermo
+   * Moritsch, Monica
+   * Mueller, Nathan
+   * Nelson, Erik
+   * Nootenboom, Chris
+   * Olwero, Nasser
+   * Ouyang, Zhiyun
+   * Papenfus, Michael
+   * Pennington, Derric
+   * Perelman, Adam
+   * Polasky, Stephen
+   * Reguero, Borja
+   * Remme, Roy
+   * Ricketts, Taylor
+   * Rogers, Lauren
+   * Ruckelshaus, Mary
+   * Silver, Jessica
+   * Schmitt, Rafael
+   * Sharp, Richard
+   * Shaw, Rebecca
+   * Soth, Emily
+   * Tallis, Heather
+   * Tam, Christine
+   * Toft, Jodie
+   * Underwood, Emma
+   * Verutes, Gregory
+   * Vogl, Adrian
+   * Wolny, Stacie
+   * Wood, Spencer
+   * Wright, Michael
+   * Wyatt, Katherine
+   * Qi, Yi
+   * Zheng, Hua
+   * Ziv, Guy
+
+Además, agradecemos la ayuda de cientos de colaboradores co-creando
+soluciones en compromisos en todo el mundo.
+
+\*Hacemos todo lo posible para mantener esta lista completa y actualizada, pero reconocemos
+es posible que hayamos perdido a alguien sin darnos cuenta. Si no ve su nombre en la lista
+anterior y cree que debería serlo, comuníquese con
+naturalcapitalproject@stanford.edu.

```

Comment:
I google translated this, I hope that's okay!

Reply:
Yep, seems fine to me!